docs(readme): add Surya Tej Nimmakayala

CodeLapseLogger opened the pull request:
Added my name and GitHub profile as a markdown link in alphabetical order, on line 1701.

Done as a sample code change to practice Pull Request creation workflow.

### Related issue
<!-- Remove this field if it doesn't fix any issue -->

### What changes does this PR do?

### Screenshots (If any)

### Questions

Panquesito7 reviewed (changes requested):
Please include your name only and revert the other changes.
Although, it seems the other changes are mostly linter changes. Do you think we should accept this in another PR, maintainers?

CodeLapseLogger commented:
Hello, I actually did make just the one change of adding the entry with my name and GitHub profile.

Like you mentioned, it seems like content spacing/alignment might have been adjusted by my local IDE (Visual Studio Code).

Would appreciate it if you have any suggestions on preventing that. I use ESLint (from Microsoft) and Prettier-Code Formatter vscode extensions. 

But, If you haven't encountered this issue before, I can certainly explore the web for a potential fix.

CodeLapseLogger commented:
I fixed the issue by installing markdownlint Visual Studio Code extension (Author: David Anson) in my local IDE and removed the extra new line after line 1. It was introduced in the earlier commit on file save, due to the absence of the linter.

Seems like the original code was pushed with markdownlint installed in the development environment. Probably a note can be added to the README.md ( another PR ? ) about this linter issue and the need to install the same to prevent unintended code changes, apart from adding one's name and GitHub profile link.

You should be able to see the new commit.

Kindly take a look and see if the branch can be merged. Do revert back to me if anything else needs to be fixed.
